From f62f09b32c98a57d92e5d99b4048355a9c894c73 Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Sun, 7 Nov 2010 11:45:52 +0000
Patch-mainline: 2.6.38
References: fate#310916
Git-commit: ae69b42a10dafe61adb016e0e52ec1e8d1ba11b4
Subject: [PATCH 1843/2588] drm/i915/ringbuffer: Be consistent in use of
 ring->size when initialising

Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
(cherry picked from commit ae69b42a10dafe61adb016e0e52ec1e8d1ba11b4)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/i915/intel_ringbuffer.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_ringbuffer.c b/drivers/gpu/drm/i915/intel_ringbuffer.c
index 53cdc3d..2c6f118 100644
--- a/drivers/gpu/drm/i915/intel_ringbuffer.c
+++ b/drivers/gpu/drm/i915/intel_ringbuffer.c
@@ -175,7 +175,7 @@ static int init_ring_common(struct intel_ring_buffer *ring)
 	}
 
 	I915_WRITE_CTL(ring,
-			((ring->gem_object->size - PAGE_SIZE) & RING_NR_PAGES)
+			((ring->size - PAGE_SIZE) & RING_NR_PAGES)
 			| RING_REPORT_64K | RING_VALID);
 
 	/* If the head is still not zero, the ring is dead */
-- 
1.7.6

